README.md

```diff
@@ -1,29 +1,59 @@
-# REST
-
+# REST API for APEL
 [![Build Status](https://travis-ci.org/apel/rest.svg?branch=dev)](https://travis-ci.org/apel/rest)
 [![Coverage Status](https://coveralls.io/repos/github/apel/rest/badge.svg?branch=dev)](https://coveralls.io/github/apel/rest?branch=dev)
 
-Experimental REST API for APEL
+The APEL project provides accounting for the Indigo DataCloud project. It is written in Python and uses MySQL.
+
+## Overview
+APEL Cloud Accounting can account for the usage of OpenNebula and OpenStack instances. Accounting "collectors" need to be installed on machines with access to the underlying Cloud infrastructure. The collectors can be found here: https://indigo-dc.gitbooks.io/indigo-datacloud-releases/content/indigo1/accounting1.html
+
+The collectors produce "Usage Records" in the APEL-Cloud v0.2 message format. Information about this format can be found here: https://wiki.egi.eu/wiki/Federated_Cloud_Accounting#Cloud_Accounting_Message_Format_for_use_with_SSM_2.0
+
+These records need to be sent as POST requests to the REST end point `.../api/v1/cloud/record`, where ... is the machine hosting the docker image. A POST request requires a X.509 certificate to authenticate the request. The hostname of the X.509 certificate must be listed as a provider here http://indigo.cloud.plgrid.pl/cmdb/service/list for the request to be authorized.
```

Comment:
- "endpoint" is usually a single word in IT service contexts.
- Should format the ellipsis as code so it matches: `...`
- "an X.509" cert.
- When you say "hostname of the cert" shouldn't that be "The hostname, which should be the same as the common name (CN) contained in the X.509 certificate, must be listed..."?